homeassistant/__main__.py

```diff
@@ -152,6 +161,49 @@ def write_pid(pid_file):
         sys.exit(1)
 
 
+def install_osx():
+    """ Setup to run via launchd on OS X """
+    with os.popen('which hass') as inp:
+      hass_path = inp.read().strip()
+
+    with os.popen('whoami') as inp:
+      user = inp.read().strip()
+
+    cwd = os.path.dirname(__file__)
+    template_path = os.path.join(cwd, 'startup', 'launchd.plist')
+
+    with open(template_path, 'r', encoding='utf-8') as inp:
+      plist = inp.read()
+
+    plist = codecs.open(template_path, 'r', 'utf-8')
+
+    plist = plist.read()
+
+    plist = plist.replace("$HASS_PATH$", hass_path)
+    plist = plist.replace("$USER$", user)
+
+    path = os.path.expanduser("~/Library/LaunchAgents/org.homeassistant.plist")
+
+    if os.path.isfile(path):
+        os.remove(path)
+
+    with codecs.open(path, 'w', 'utf-8') as outp:
```

Comment:
Can you convert this to the `open` method so the codecs import can be removed too?